packages/nimble-components/src/number-field/specs/IxD.md

```diff
@@ -0,0 +1,160 @@
+| Placeholder         | Shows an example of the data required in the field or a description of the purpose of the field. |
+| Numeric value       | Current control value.                                                                           |
+| Increment/Decrement | Changes numeric value by the step size.                                                          |
+| Error status icon   | Appears when a user has entered invalid data in a field.                                         |
+| Error text          | Appears when a user has entered invalid data in a field.                                         |
+
+### Related Components
+
+- [Number Field](https://nimble.ni.dev/storybook/index.html?path=/docs/components-number-field--docs)
+
+## Design
+
+### Configuration
+
+Proposed attributes:
+
+| Attribute               | Description                                                                                                               |
+| ----------------------- | ------------------------------------------------------------------------------------------------------------------------- |
+| maximum-fraction-digits | Maximum number of digits after the decimal separator.                                                                     |
+| minimum-fraction-digits | Minimum number of fraction digits. Values with fewer digits will be right-padded with zeros to meet the specified length. |
+| min                     | The lowest permitted value.                                                                                               |
+| max                     | The greatest permitted value.                                                                                             |
+
+### Behavior
+
+#### Mouse Interactions
+
+Hovering over the field input indicates interactivity.
+
+Clicking on the field input focuses the input and places a caret. If no value
+exists, the caret is placed at the far left. Otherwise, it is placed where
+clicked.
+
+Double-clicking within the field input selects the closest contiguous
+characters.
+
+Triple-clicking within the field input selects all characters.
+
+![mouse interactions](./spec-images/localized-number-input-cursor-click.png)
+
+#### Keyboard Interactions
+
+| Key                       | Description                                                                                                                                      |
+| ------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `ENTER`/`ESC`             | No action.                                                                                                                                       |
+| `UP ARROW`                | (If `hide-step = false`) Increment the number by the step value. (If `hide-step = true`) Moves the caret to the beginning of the entered number. |
```

Comment:
This would be a change in behavior (same for down arrow). Currently, even if you `hide-step`, up/down increment/decrement the value.